src/model/ControllerOutdoorAir.hpp

```diff
@@ -105,10 +108,12 @@ namespace model {
     bool setMinimumLimitType(const std::string& value);
 
     boost::optional<bool> getHighHumidityControl() const;
-    bool setHighHumidityControl(bool val);
+    OS_DEPRECATED(3, 8, 0) bool setHighHumidityControl(bool val);
 
-    //Zone getHumidistatControlZone() const;
-    //bool setHumidistatControlZone(Zone z)
+    boost::optional<ThermalZone> getHumidistatControlZone() const;
+    boost::optional<ThermalZone> humidistatControlZone() const;
```

Comment:
```suggestion
    boost::optional<ThermalZone> humidistatControlZone() const;
```

I would say, let's not introduce two getters that we'll have to maintain.

I know the rest of the getters are `get<Value>()`, but I hate it, so I'd rather add a regular `<value>()` one if I had to pick.

This is just a suggestion, if you feel strongly about having both to maintain file-consistency.